package/src/skia/types/Image/Image.ts

```diff
@@ -94,6 +94,12 @@ export interface SkImage extends SkJSIInstance<"Image"> {
   */
   encodeToBase64(fmt?: ImageFormat, quality?: number): string;
 
+  /** Read Image RGBA pixels, returning result as a UInt8Array.
+   * 
+   * @return  Uint8Array with data
+   */
+  readPixels(): Uint8Array;
```

Comment:
I had implement the API like the CanvasKit signature.

For the `dest` I use TypedArray instead MallocObj.
Because there no other use case need to store the shared buffer on native.